modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -3,150 +3,63 @@
 
 == Introduction
 
-Federated Cloud Sharing is now managed by the Federation app (9.0+), and
-is now called Federation sharing. When you enable the Federation app you
-can easily and securely link file shares between ownCloud servers, in
-effect creating a "cloud" of ownCloud installations.
+Federated Cloud Sharing is managed by the Federation app. When you enable the Federation app you can easily and securely link file shares between ownCloud servers, in effect creating a "cloud" of ownCloud installations.
 
 [IMPORTANT]
 ====
 For security reasons federated sharing **strictly requires HTTPS (SSL/TLS)**.
```

Comment:
It would actually work even with HTTP when the config.php option `'sharing.federation.allowHttpFallback'` is set to `true`. Not sure it is worth mentioning this, but this option is officially documented in our docs.

Reply:
Have you tested this?

I remember me trying to set it up without certificates, in http mode, and I failed. Did you manage to get it to work?

Reply:
Also maybe it's better to have in the docs the "save" method, so people don't come to us complaining the have a security flaw just because they followed the docs :)

Reply:
Yes, i tested this option successfully.

Reply:
added